paymentsheet/src/test/java/com/stripe/android/lpmfoundations/paymentmethod/definitions/CardDefinitionTest.kt

```diff
@@ -87,4 +93,38 @@ class CardDefinitionTest {
         assertThat(formElements[0].identifier.v1).isEqualTo("card_details")
         assertThat(formElements[1].identifier.v1).isEqualTo("save_for_future_use")
     }
+
+    @Test
+    fun `createFormElements returns link_form`() {
```

Comment:
We already implicitly test it's not included in the other tests by the size, correct?

Reply:
Correct!